### ISSUE TITLE
nvim-tree help window displays behind floating file selector window since Neovim 0.10.0

### ISSUE THREAD

**Issue by proofer** [neovide/neovide#2624]
**Describe the bug**
nvim-tree displays a window containing a file selector; when configured to be floating and offset from the upper left corner:  typing `g?` displays another "help" window listing nvim-tree's keymaps. Prior to Neovim 0.10.0, this help window appeared alone. Since 0.10.0 the help window appears *behind* the file selector window and is thus unusable. See context below for conditions.

**To Reproduce**
(1) install nvim-tree; e.g., with lazy.nvim, this ~minimal config:
```
return {
    'nvim-tree/nvim-tree.lua',
    version = '*',
    lazy = false,
    config = function()
        require('nvim-tree').setup({
            view = {
                float = {
                    enable = true,
                    open_win_config = {
                        width = 50,
                        height = 50,
                       -- the floating aspect: not row=0, col=0:
                        row = 8,
                        col = 8,
                    }, }, }, })
    end,
}
```
(2) open Neovide
(3) type: `:NvimTreeOpen`
(4) type: `g?`
(5) observe that the "help" window opens *behind* the file-selector window and therefore its content is mostly obscured.

**Expected behavior**
An nvim-tree help window opens at the upper left corner in a plane *in front* of the file-selector window.

**Screenshots**
(On request)

**Desktop (please complete the following information):**
 - OS: MacOS 14.5
 - Neovide Version 0.13.1
 - Neovim Version 0.10.0

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
(N/A)

**Additional context**
The file-selector window must be floating -- offset from the upper left corner -- to induce the issue.
The problem does *not* occur when nvim is opened in iTerm2: the "help" window opens *in front* of the file-selector window.
Prior to 0.10.0, in Neovide the "help" window *replaced* the file-selector window, and there was no way to dismiss only it and refocus on the file-selector window. Instead, I had to type `q` which required re-opening the file-selector window.

**Comment by fredizzimo** [comment]
This will be fixed by
* https://github.com/neovide/neovide/pull/2612

**Comment by proofer** [comment]
> This will be fixed by
> 
> * [fix: emulate Neovim layer drawing #2612](https://github.com/neovide/neovide/pull/2612)

Great! (Thanks for waiting by your terminal for me to post this issue... :)
